### ISSUE TITLE
Registering new account causes lots of SQL errors

### ISSUE THREAD

**Issue by markasoftware** [wlox/wlox#84]
When registering a new account, I get SQL errors on the insert into the site_users table. It's because lots of the columns in site_users are not written to in the command, but do not have default values either, for example `first_name` and `last_name`. These are market as `not null` so it fails. This is on the mbassan fork.

**Comment by mbassan** [comment]
Did you switch the database file to the one from the mbassan fork?

**Comment by markasoftware** [comment]
Yep. I restored from the mbassan sql dump. You can see on line 6259 where the site_users table is created in the sql file, it creates the `first_name` column with not null. And in line 544 or so in `User.php` in api/lib, it inserts into the database without a `first_name` column value specified. And it's not just the `first_name` column, there are several others as well.

**Comment by mbassan** [comment]
Will be solved now in mbassan fork.
